macros/SVGAttr.ejs

```diff
@@ -3,7 +3,6 @@
 var slug = mdn.localString({
     "en-US": "Attribute",
     "de"   : "Attribut",
-    "fr"   : "Attributs",
     "pl"   : "Atrybut"
```

Comment:
[`/pl/docs/Web/SVG/Attribute`](https://developer.mozilla.org/pl/docs/Web/SVG/Attribute) isn’t translated and [`Atrybut`](https://developer.mozilla.org/pl/docs/Web/SVG/Atrybut) returns a 404.